repairnator/pipeline-dockerimage4Bears/build_repairnator4Bears.sh

```diff
@@ -0,0 +1,20 @@
+#!/bin/bash
+
+set -e
+
+cd /root
+mkdir github
+cd github
+git clone --recursive https://github.com/fermadeiral/repairnator.git
```

Comment:
I think the URL is wrong there.

Reply:
As I already told, this is temporary (I need this to test my changes). I will change it just before the end of this PR.

Reply:
Ok :)

Reply:
I will wait for your signal to undo this before merging.